### PR TITLE
CI: Try compile wasm to aot more times

### PR DESCRIPTION
**Pull request by no1wudi**

### CODE REVIEW

**Comment by no1wudi** [comment]
Refer to this error: https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/10385477773/job/28754889489

File: tests/wamr-test-suites/spec-test-script/runtest.py [hunk 1]
```diff
@@ -1308,8 +1308,8 @@ def test_assert_with_exception(form, wast_tempfile, wasm_tempfile, aot_tempfile,
     if test_aot:
         aot_tempfile = create_tmp_file(".aot")
         # could be potientially compiled to aot
-        # with the future following call test_assert_xxx, 
-        # add them to temp_file_repo now even if no actual following file, 
+        # with the future following call test_assert_xxx,
```
Comment (no1wudi) [inline tests/wamr-test-suites/spec-test-script/runtest.py:1311]: These changes is from my editor, it trim the tail space automatically

**Comment by no1wudi** [comment]
> are you referring to this error?

For this error:
```
Compiling '/tmp/M.wasm' to '/tmp/M.aot'
Running: /__w/wasm-micro-runtime/wasm-micro-runtime/apps/interpreters/wamr/wamr/tests/wamr-test-suites/workspace/../../../wamr-compiler/build/wamrc --target=riscv64 --target-abi=lp64 --cpu=generic-rv64 --cpu-features=+m,+a,+c --size-level=1 --disable-simd --enable-gc --enable-tail-call --disable-llvm-lto --bounds-checks=1 -o /tmp/M.aot /tmp/M.wasm
Compiling '/tmp/M.wasm' to '/tmp/M.aot'
Running: /__w/wasm-micro-runtime/wasm-micro-runtime/apps/interpreters/wamr/wamr/tests/wamr-test-suites/workspace/../../../wamr-compiler/build/wamrc --target=riscv64 --target-abi=lp64 --cpu=generic-rv64 --cpu-features=+m,+a,+c --size-level=1 --disable-simd --enable-gc --enable-tail-call --disable-llvm-lto --bounds-checks=1 -o /tmp/M.aot /tmp/M.wasm
Compiling '/tmp/M.wasm' to '/tmp/M.aot'
Running: /__w/wasm-micro-runtime/wasm-micro-runtime/apps/interpreters/wamr/wamr/tests/wamr-test-suites/workspace/../../../wamr-compiler/build/wamrc --target=riscv64 --target-abi=lp64 --cpu=generic-rv64 --cpu-features=+m,+a,+c --size-level=1 --disable-simd --enable-gc --enable-tail-call --disable-llvm-lto --bounds-checks=1 -o /tmp/M.aot /tmp/M.wasm
THE FINAL EXCEPTION IS compile wasm to aot failed
```

**Comment by no1wudi** [comment]
Emm, by reading the error log, the root case seems be:
```
Compiling '/tmp/M.wasm' to '/tmp/M.aot'
```
The filename is too simple to conflict with others.

**Comment by no1wudi** [comment]
@wenyongh Do you have any idea for this issue?

**Comment by wenyongh** [comment]
> @wenyongh Do you have any idea for this issue?

Not know what the issue is, why fail to compile M.wasm to M.aot? Is it because that multiple threads are using wamrc to compile the same M.wasm to M.aot?

**Comment by no1wudi** [comment]
> Not know what the issue is, why fail to compile M.wasm to M.aot? Is it because that multiple threads are using wamrc to compile the same M.wasm to M.aot?

It is possible, and it appears that this issue cannot be resolved through the current modifications.

**Pull request closed without merging**